docs/DeveloperGuide.md

```diff
@@ -262,29 +262,50 @@ _{Explain here how the data archiving feature will be implemented}_
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
-
-*{More to be added}*
+| Priority | As a …​            | I want to …​                                                                        | So that I can…​                                                                 |
+|----------|-------------------|------------------------------------------------------------------------------------|---------------------------------------------------------------------------------|
+| `* * *`  | New user          | easily download and launch FINDvisor                                               | quickly start managing my client information                                    |
+| `* * *`  | New user          | know what are the available functionalities of FINDvisor                           |                                                                                 |
+| `* * *`  | New user          | know how to operate the basic functionalities of FINDvisor within the app         |                                                                                 |
+| `* * *`  | Financial Advisor | add contacts of my clients                                                         | keep a record of my clients' contact information.                                |
+| `* * *`  | Financial Advisor | find contacts of my clients                                                        | find information on a specific client.                                          |
+| `* * *`  | Financial Advisor | update client's contact information                                                | not need to delete and create new contact information.                           |
+| `* * *`  | Financial Advisor | remove contact information                                                         | reduce clutter in contact list with clients I have severed ties with.           |
```

Comment:
I think we should remove the punctuation at the end to keep it consistent? Just a nit.